ghcide/src/Development/IDE/Main.hs

```diff
@@ -218,6 +218,7 @@ defaultMain Arguments{..} = do
                 config <- LSP.runLspT env LSP.getConfig
                 let options = (argsIdeOptions config sessionLoader)
                             { optReportProgress = clientSupportsProgress caps
+                            , optModifyDynFlags = pluginModifyDynflags plugins
```

Comment:
This throws away the callers `optModifyDynFlags`, which is no good. Please append to whatever is there instead.